packages/fx-core/resource/package.nls.json

```diff
@@ -1163,6 +1167,14 @@
   "driver.deploy.deployToAzureFunctionsDescription": "deploy the project to the Azure Functions.",
   "driver.deploy.deployToAzureStorageDescription": "deploy the project to the Azure Storages.",
   "driver.deploy.enableStaticWebsiteInAzureStorageDescription": "enable static website setting in Azure Storage.",
+  "driver.deploy.getStorageContainerError": "Failed to get storage container %s.",
+  "driver.deploy.checkContainerStaticWebsiteError": "Failed to check if the container %s enabled static website option.",
+  "driver.deploy.checkContainerStaticWebsiteRemoteError": "Failed to check if the container %s enabled static website option due to remote service error.",
+  "driver.deploy.enableStaticWebsiteError": "Failed to enable static website option for the container %s.",
```

Comment:
same as comment for next line, please double check whether brand name is aligned.